.pre-commit-config.yaml

```diff
@@ -1,16 +1,17 @@
-# To enable this pre-commit hook (in this case, pre-push hook), run:
+# To enable this pre-commit hook run:
 # `python3 -m pip install pre-commit` or `brew install pre-commit`
```

Comment:
Using `python3 -m pip install pre-commit` did not work for me on my system (Ubuntu 18.04).

Using `pip install pre-commit` worked.  I found this here: https://pre-commit.com/#install

Aside from that, everything seems to be working fine.

Reply:
Ultimately this should be added to the readme but not until it's working with flake8 and black. I added `pip install pre-commit` here for now though 👍